brainscore/benchmarks/search.py

```diff
@@ -85,3 +85,84 @@ def ceiling(self):
 
     def get_raw_data(self):
         return self.cumm_perf, self.saccades
+
+KlabZhang2018search_assembly = ['klab.Zhang2018search_waldo', 'klab.Zhang2018search_naturaldesign']
+KlabZhang2018search_assembly_ceil = [1, 1]
+
+class KlabZhang2018VisualSearch(Benchmark):
+    def __init__(self, ceil_score=None, assembly_name=None):
+        if assembly_name is None:
+            self._logger.info("## No assembly name given, therefore selecting random assembly from 'waldo' or 'naturaldesign'...\n")
+            i = np.random.choice(2)
+            assembly_name = KlabZhang2018search_assembly[i]
+            ceil_score = KlabZhang2018search_assembly_ceil[i]

```

Comment:
Shouldn't we just have one benchmark for each of the two assemblies rather than selecting randomly?

Reply:
again it's the outdated version. please check the latest one.